libs/api/domains/endorsement-system/src/lib/models/endorsementList.model.ts

```diff
@@ -30,6 +30,7 @@ export class EndorsementList {
   @Field(() => [ValidationRule])
   validationRules!: ValidationRule[]
 
+  // Remove this field when changes have been made to frontend so that we dont expose the owners ssn
```

Comment:
It might be better to update the frontend in this PR as well.
Since that confirm that this flow works in frontend